diff --git a/node_modules/rollup/dist/rollup.js b/node_modules/rollup/dist/rollup.js
index afe921a..f316ef5 100644
--- a/node_modules/rollup/dist/rollup.js
+++ b/node_modules/rollup/dist/rollup.js
@@ -13477,7 +13477,7 @@ function rollup(rawInputOptions) {
             return __awaiter(this, void 0, void 0, function* () {
                 timeStart('GENERATE', 1);
                 const assetFileNames = outputOptions.assetFileNames || 'assets/[name]-[hash][extname]';
-                const inputBase = commondir(getAbsoluteEntryModulePaths(chunks));
+                const inputBase = process.cwd();
                 const outputBundleWithPlaceholders = Object.create(null);
                 outputPluginDriver.setOutputBundle(outputBundleWithPlaceholders, assetFileNames);
                 let outputBundle;
