diff --git a/node_modules/rollup/dist/rollup.js b/node_modules/rollup/dist/rollup.js
index 364c40d..35e3e3e 100644
--- a/node_modules/rollup/dist/rollup.js
+++ b/node_modules/rollup/dist/rollup.js
@@ -13445,7 +13445,7 @@ function rollup(rawInputOptions) {
                 const assetFileNames = outputOptions.assetFileNames || 'assets/[name]-[hash][extname]';
                 const outputBundleWithPlaceholders = Object.create(null);
                 let outputBundle;
-                const inputBase = commondir(getAbsoluteEntryModulePaths(chunks));
+                const inputBase = process.cwd();
                 graph.pluginDriver.startOutput(outputBundleWithPlaceholders, assetFileNames);
                 try {
                     yield graph.pluginDriver.hookParallel('renderStart', []);
